src/code/dialogs/markerList.js

```diff
@@ -27,6 +27,8 @@ const MarkerList = OperationChecklistDialog.extend({
       this.closeDialog();
     };
 
+    await this.sortable.done;
```

Comment:
done is a getter, which isn't async. does this work?

Reply:
done returns _done that is either a promise or  boolean
also, you can always await a constant expression